src/lib/format.js

```diff
@@ -138,7 +138,9 @@ export function entityFromContext(ctx, errors, id, args) {
     for (let i = 0, trait; (trait = entity.traits[i]); i++) {
       const attr = ctx.format(trait, args, errors);
       if (attr !== null) {
-        formatted.attrs[trait.key.name] = attr;
+        let key =
+          trait.key.ns ? `${trait.key.ns}/${trait.key.name}` : trait.key.name;
+        formatted.attrs[key] = attr;
```

Comment:
This looks like a step backwards.  First the parser parses the namespaces and then this code serializes them back, just to `split('/')` again in `overlayElement`.

I'd prefer something more semantic:

 - either store namespaced attributes in `formatted[Symbol.for(trait.key.ns)][trait.key.name]`, or
 - make `formatted.attrs ` an array and push `[namespace, key, value]` tuples to it.

Also, we should ignore all traits which don't have any namespaces here as they are local to the language and shouldn't be exposed to the app.  You could check for `trait.key.ns` even before calling `ctx.format`.